code/modules/mod/mod_pai.dm

```diff
@@ -49,7 +49,9 @@
 #define PAI_FALL_TIME 1 SECONDS
 
 /obj/item/mod/control/relaymove(mob/user, direction)
-	if((!active && wearer) || !cell || cell.charge < CELL_PER_STEP  || user != mod_pai || !COOLDOWN_FINISHED(src, cooldown_mod_move) || (wearer?.pulledby?.grab_state))
+	if((!active && wearer) || !cell || cell.charge < CELL_PER_STEP  || user != mod_pai || !COOLDOWN_FINISHED(src, cooldown_mod_move))
+		return FALSE
+	if(wearer && (wearer.pulledby.grab_state || IS_DEAD_OR_INCAP(wearer)))
```

Comment:
IS_DEAD_OR_INCAP is for ai controllers, everything that would make a human have stat would also incapacitate them